### DISCUSSION TITLE
Release cadence for Helm chart updates

### DISCUSSION THREAD

**Discussion by kumachop2** [signalfx/splunk-otel-collector-chart#783] [General] [answered]
Would like to know the release calendar for  splunk-otel-collector-chart as well as splunk-otel-collector.

**Comment by jvoravong** [comment, marked as answer]
At this time, we don't commit to a specific release schedule. Our releases typically follow the releases of the related upstream OpenTelemetry projects ([collector core](https://github.com/open-telemetry/opentelemetry-collector), [collector contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib), [helm charts](https://github.com/open-telemetry/opentelemetry-helm-charts)).  Upstream projects usually release about every 2 weeks.
